src/libasf_raster/diffimage.c

```diff
@@ -5868,7 +5096,11 @@ void export_jpeg_to_asf_img(char *inFile, char *outfile,
 
   // Write out the (very very simple) metadata file ...
   // only contains line_count, sample_count, and data_type
-  make_generic_meta(fft_meta_file, jpg.height, jpg.width, jpg.data_type);
+  char **bands;
```

Comment:
In this and the other get_band_names() calls, the returned "bands" needs to be free()d